File: CuartoPunto/Programa.c
#include <stdio.h>
#include <time.h>

int fib(int n) {
    if (n <= 1) return n;
    return fib(n-1) + fib(n-2);
}

int main() {
    int n = 48;
    clock_t inicio = clock();
    
    int resultado = fib(n);
    
    clock_t fin = clock();
    double tiempo = (double)(fin - inicio) / CLOCKS_PER_SEC;
    
    printf("Fibonacci(%d) = %d\n", n, resultado);
    printf("Tiempo en C: %f segundos\n", tiempo);
    return 0;
}

File: CuartoPunto/Programa.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n-1) + fib(n-2)

n = 48
inicio = time.time()

resultado = fib(n)

fin = time.time()
print(f"Fibonacci({n}) = {resultado}")
print(f"Tiempo en Python: {fin - inicio} segundos")